### PR TITLE
Add return value when sending a notification

### PR DESCRIPTION
**Pull request by alexduf**
- Add traces
 - return 201
 - test for 201 in the azure client

### CODE REVIEW

File: common/src/main/scala/azure/NotificationHubClient.scala [hunk 1]
```diff
@@ -53,7 +53,7 @@ class NotificationHubClient(notificationHubConnection: NotificationHubConnection
       .withHeaders(serviceBusTags: _*)
       .post(azureWindowsPush.body)
       .map { response =>
-        if (response.status == 200)
+        if (response.status >= 200 || response.status < 300)
```
Comment (davidfurey) [inline common/src/main/scala/azure/NotificationHubClient.scala]: I think you meant if (response.status >= 200 && response.status < 300)

File: common/src/main/scala/models/PushResult.scala [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package models
+
+import java.util.UUID
+
+import play.api.libs.json.Json
+
+case class PushResult(id: UUID)
+
+object PushResult {
+  import JsonUtils._
```
Comment (davidfurey) [inline common/src/main/scala/models/PushResult.scala]: What does this import do?

Comment (alexduf) [inline common/src/main/scala/models/PushResult.scala]: A json formatter for UUID

File: common/src/main/scala/azure/NotificationHubClient.scala [hunk 3]
```diff
@@ -53,7 +53,7 @@ class NotificationHubClient(notificationHubConnection: NotificationHubConnection
       .withHeaders(serviceBusTags: _*)
       .post(azureWindowsPush.body)
       .map { response =>
-        if (response.status == 200)
+        if (response.status >= 200 && response.status < 300)
```
Comment (agulowaty) [inline common/src/main/scala/azure/NotificationHubClient.scala]: `(200 until 300).contains(response.status)` could be more clear I think

File: common/src/main/scala/models/PushResult.scala (hunk 2 above)
Comment (alexduf) [inline common/src/main/scala/models/PushResult.scala]: Nope, I'm wrong, it's useless

Comment (agulowaty) [inline common/src/main/scala/models/PushResult.scala]: maybe we should make it more explicit ?

File: notification/app/notification/controllers/Main.scala [hunk 4]
```diff
@@ -39,13 +40,15 @@ final class Main @Inject()(
       case \/-(report) =>
         notificationReportRepository.store(report) map {
           case \/-(_) =>
-            Ok("Ok")
+            Logger.info(s"Notification was sent: $push")
```
Comment (agulowaty) [inline notification/app/notification/controllers/Main.scala]: this will output whole Push object to info logger ?

Comment (alexduf) [inline notification/app/notification/controllers/Main.scala]: Yes, I think that could be useful for audit. We only keep 7 days of logs anyway.

**Comment by alexduf** [comment]
Any one to have another quick look ?

**Comment by agulowaty** [comment]
:+1:

**Pull request merged**